### ISSUE TITLE
prevent anchor behavior

### ISSUE THREAD

**Issue by rstacruz** [zengabor/zenscroll#5]
I'd like to use `zenscroll.intoView()` but I don't really want the smooth scrolling to happen when I click on anchor links. Is there a way to disable that behavior?

**Comment by zengabor** [comment]
The funny thing is I just released 2.0.0 a few minutes ago which removed the possibility of disabling that behavior. :) I did it because it makes the code simpler and shorter, and thought that no-one would want to disable it anyway.

You can easily disable it by commenting out the following line:

```
// win.addEventListener("click", internalLinkHandler, false)
```

Or I could add back something like `zenscroll.stop()`

**Comment by zengabor** [comment]
Alright, just released version 2.1.0 for you. Set `window.noZensmooth` to a non-false value and the event handler for automatic smoothing will not be installed but you can still use everything, like `zenscroll.intoView()`, etc.

``` html
    ...
    <script>window.noZensmooth = true</script>
    <script src="zenscroll-min.js"></script>
</body>
```

(I consider this a rare scenario that’s why I keep the default behavior of installing the event handler.)

**Comment by rstacruz** [comment]
Neat. Thanks!
